pro/src/components/hooks/useAnalytics.ts

```diff
@@ -31,18 +25,7 @@ export interface IUseAnalyticsReturn {
 }
 
 const useAnalytics = (): IUseAnalyticsReturn => {
-  const app = useRef<firebase.FirebaseApp>()
-
-  if (!app.current) {
-    app.current = firebase.initializeApp(firebaseConfig)
-    initializeAnalytics(app.current, {
-      config: {
-        send_page_view: false,
-      },
-    })
-  }
```

Comment:
J'aurais du le mettre dans le 1er commit, mais `initializeAnalytics` ici me levait une erreur 
```Analytics: initializeAnalytics() cannot be called again with different options than those it was initially called with. It can be called again with the same options to return the existing instance, or getAnalytics() can be used to get a reference to the already-intialized instance. (analytics/already-initialized).```
Est-ce que l'extraire est une bonne idée? J'ai l'impression que c'est initialisé qu'une seule fois avec ce changement